rs/sns/governance/src/topics.rs

```diff
@@ -0,0 +1,200 @@
+            content: BuiltInFunctions {
+                native_functions: vec![
+                    UPGRADE_SNS_CONTROLLED_CANISTER,
+                    REGISTER_DAPP_CANISTERS,
+                    MANAGE_DAPP_CANISTER_SETTINGS,
+                ],
+            },
+            is_critical: false,
+        },
+        TopicInfo::<BuiltInFunctions> {
+            topic: Topic::ApplicationBusinessLogic,
+            name: "Application Business Logic".to_string(),
+            description: "Proposals that are custom to what the governed dapp requires.".to_string(),
+            content: BuiltInFunctions {
+                native_functions: vec![],
+            },
+            is_critical: false,
+        },
+        TopicInfo::<BuiltInFunctions> {
+            topic: Topic::Governance,
+            name: "Governance".to_string(),
+            description: "Proposals that represent community polls or other forms of community opinion but don’t have any immediate effect in terms of code changes.".to_string(),
+            content: BuiltInFunctions {
+                native_functions: vec![MOTION],
+            },
+            is_critical: false,
+        },
+        TopicInfo::<BuiltInFunctions> {
+            topic: Topic::TreasuryAssetManagement,
+            name: "Treasury & asset management".to_string(),
+            description: "Proposals to move and manage assets that are DAO-owned, including tokens in the treasury, tokens in liquidity pools, or DAO-owned neurons.".to_string(),
+            content: BuiltInFunctions {
+                native_functions: vec![
+                    TRANSFER_SNS_TREASURY_FUNDS,
+                    MINT_SNS_TOKENS,
+                ],
+            },
+            is_critical: true,
+        },
+        TopicInfo::<BuiltInFunctions> {
+            topic: Topic::CriticalDappOperations,
+            name: "Critical Dapp Operations".to_string(),
+            description: "Proposals to execute critical operations on dapps, such as adding or removing dapps from the SNS, or executing custom logic on dapps.".to_string(),
+            content: BuiltInFunctions {
+                native_functions: vec![
+                    DEREGISTER_DAPP_CANISTERS,
+                    ADD_GENERIC_NERVOUS_SYSTEM_FUNCTION,
+                    REMOVE_GENERIC_NERVOUS_SYSTEM_FUNCTION,
+                ],
+            },
+            is_critical: true,
+        },
+    ]
+}
+
+impl Governance {
+    pub fn list_topics(&self) -> ListTopicsResponse {
+        let mut uncategorized_functions = vec![];
+
+        let topic_id_to_functions: HashMap<u64, NervousSystemFunction> =
+            native_action_ids::native_functions()
+                .into_iter()
+                .map(|function| (function.id, function))
+                .collect();
+        let generic_functions = self
```

Comment:
```suggestion
        let custom_functions = self
```
Please make the naming consistent throughout !